pkg/apis/apiserverinternal/validation/validation.go

```diff
@@ -34,6 +34,16 @@ func ValidateStorageVersion(sv *apiserverinternal.StorageVersion) field.ErrorLis
 	return allErrs
 }
 
+// ValidateStorageVersionUpdate tests if an update to a StorageVersion is valid.
+func ValidateStorageVersionUpdate(sv, oldSV *apiserverinternal.StorageVersion) field.ErrorList {
+	return ValidateStorageVersion(sv)
```

Comment:
this will fail an update if the status (which cannot be changed in a main resource update) is invalid... that doesn't seem useful